lib/modules/manager/unity3d/readme.md

```diff
@@ -0,0 +1,10 @@
+Handles version numbers inside `ProjectSettings/ProjectVersion.txt` files used by the Unity game engine.
```

Comment:
I see two ways to read this:

- Renovate can read `ProjectSettings.txt` and `ProjectVersion.txt` files
- Renovate can read a `ProjectVersion.txt` file _if_ its in the `ProjectSettings` directory

I don't know which interpretation is correct, as I don't use Unity.

Please update the text so it explains precisely what Renovate can update. 🙂